addon/styles/tip.less

```diff
@@ -0,0 +1,74 @@
+@import url('https://fonts.googleapis.com/css?family=Lato:300,400,700');
```

Comment:
Looks like we have multiple imports like this in our styles. Maybe we could just include one in `variables.less` or `addon.less`?

Reply:
👍